### ISSUE TITLE
If a BaseModel has a __call__ method, the IDE warns that the call method only accepts kwargs

### ISSUE THREAD

**Issue by matthiasmindee** [koxudaxi/pydantic-pycharm-plugin#651]
**Describe the bug**
If you have a class that inherits from pydantic's `BaseModel` and this class is callable, then the plugin (via the IDE) will complain that the `__call__` method only accepts kwargs even though it explicitely accepts both args and kwargs. Without the plugin the IDE raises no warning, with it it does. 

**To Reproduce**
This code reproduces the issue:
```python
from pydantic import BaseModel


class Foo(BaseModel):
    bar: int
    baz: str

    def __call__(self, *args, **kwargs):
        return 1


if __name__ == '__main__':
    foo = Foo(bar=1, baz="3")
    i = foo(1)
```

**Expected behavior**
The IDE should probably hint the arguments of the `__call__` method, or at least not complain that the arguments are kwargs only.

**Screenshots**
![image](https://user-images.githubusercontent.com/92733074/220378607-b053b0e7-d52f-45af-aae2-2fb4e42ba52c.png)

**Environments:**
 - IDE: PyCharm 2022.3.2 (Professional Edition) Build #PY-223.8617.48, built on January 24, 2023
 - OS: Ubuntu 22.04
 - Pydantic Version 1.10.4
 - Plugin version 0.3.17

**Comment by matthiasmindee** [comment]
@koxudaxi Thanks for such a quick fix !

**Comment by koxudaxi** [comment]
@matthiasmindee   
I have released a new version [v0.4.0](https://github.com/koxudaxi/pydantic-pycharm-plugin/releases/tag/v0.4.0) in [the market](https://plugins.jetbrains.com/plugin/12861-pydantic/versions/stable/300885)
The version supports `__call__`.
Thank you.

**Comment by matthiasmindee** [comment]
Thanks a lot, I'll test that now !
